output.go

```diff
@@ -43,7 +43,7 @@ type OutputAccount struct {
 	// that should be updated in the account storage.
 	// Please note that it is likely that not all existing
 	// account storage keys show up here.
```

Comment:
Could you please also add some documentation in the comments, as to what the map keys represent?

Reply:
Updated the comment.